apollo/src/resolvers/Project.js

```diff
@@ -1,56 +1,76 @@
+// @ts-check
+
 // Resolves all relational fields on type Project
 // where the name of the function is an exact match to the field
 
-const product = (parent, args, context) => {
+/**
+ * @param { import('../context').ApolloContext } context
+ * @returns { import('../generated/prisma-client').ProductPromise }
+ */
+const product = (parent, _, context) => {
   const res = context.prisma.project({ id: parent.id }).product();
 
   return res;
```

Comment:
Should we replace all of these with a single line similar to `apollo/src/resolvers/Program.js`?
```suggestion
  return context.prisma.project({ id: parent.id }).product();
```